deployment/README.md

```diff
@@ -38,34 +35,15 @@ Deploying your App on the Cloud
 Microsoft.Spark.Worker is a backend component that lives on the individual worker nodes of your Spark cluster. When you want to execute a C# UDF (user-defined function), Spark needs to understand how to launch the .NET CLR to execute this UDF. Microsoft.Spark.Worker provides a collection of classes to Spark that enable this functionality.
 
 ## Microsoft.Spark.Worker
-1. Publish Microsoft.Spark.Worker as self-contained.
-```shell
-# For example, you can run the following on Linux.
-foo@bar:~/dotnet/spark/src/csharp/Microsoft.Spark.Worker$ dotnet publish -c Release -f netcoreapp2.1 -r ubuntu.16.04-x64
-```
-> **Note**: Ensure that the correct [dotnet Runtime Identifier](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) is used for your cluster.
-
-2. Produce `worker.tgz` for the published files.
-```shell
-# For example, you can run the following on Linux using `tar`.
-foo@bar:~/dotnet/spark/src/csharp/Microsoft.Spark.Worker$ tar czvf worker.tgz -C bin/Release/netcoreapp2.1/ubuntu.16.04-x64/publish/ .
-```
+1. Select a [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases) Linux netcoreapp release to be deployed on your cluster.
 
-3. Upload `worker.tgz` and [install-worker.sh](install-worker.sh) to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to.
+2. Upload `Microsoft.Spark.Worker.netcoreapp<dotnet_version>.linux-x64-<spark_dotnet_version>.tar.gz` and [install-worker.sh](install-worker.sh) to a distributed file system (e.g., HDFS, WASB, ADLS, S3) that your cluster has access to.
 
```

Comment:
Something weird in L164 sentence.

Reply:
What's wrong with L164?

Reply:
Search for this text: "Configure and your Data Source and mount it" Basically, L164 got changed once you committed :) The sentence formation doesn't look right.

Reply:
I cannot point exactly at that line since it is not part of the change set.